client/package.json

```diff
@@ -52,6 +52,7 @@
     "@types/cookie": "0.5.1",
     "@types/jest": "29.0.0",
     "@types/lodash": "4.14.184",
+    "@types/next": "9.0.0",
```

Comment:
Not sure you need it.

https://www.npmjs.com/package/@types/next

<img width="1316" alt="image" src="https://user-images.githubusercontent.com/618807/191906932-58fd58e3-6f44-494c-a65b-4e3787306a4c.png">

Reply:
I've got bunch of warnings about lack of types declarations for Next components (e.g. `Header`). If we're okay with this I can remove it :)

Reply:
hm, it seems that it was minor local issue, removed it